pages/treeherder.py

```diff
@@ -91,15 +125,27 @@ def unclassified_failure_count(self):
     def clear_filter(self):
         self.selenium.find_element(*self._clear_filter_locator).click()
 
+    def clear_filter_by_shortcut(self):
+        el = self.find_element(*self._result_sets_locator)
+        el.send_keys(Keys.CONTROL + Keys.SHIFT + 'f')
```

Comment:
I would probably make an exception for body and not create a locator. The body is not something that's likely to change:

```python
self.find_element(By.CSS_SELECTOR, 'body').send_keys(Keys.CONTROL + Keys.SHIFT + 'f')
```